README.md

```diff
@@ -37,6 +37,15 @@ or if sqlite support is required:
 
     TAGS="bindata sqlite sqlite_unlock_notify" make build
 
+The `build` target is split into two sub-targets:
+
+- `make backend` which requires [Go 1.11](https://golang.org/dl/) or greater.
+- `make frontend` which requires [Node.js 10.0.0](https://nodejs.org/en/download/) or greater.
+
+If pre-built frontend files are present it is possible to only build the backend:
+
+		TAGS="bindata" make backend
```

Comment:
Could we still use `TAGS="bindata" make backend`, that means, we only check node if we need build frontend. And also `make build` should know if we need to build frontend.

Reply:
It's not always determinable if frontend needs rebuilding. For tarball sources, the timestamp of files depend on extraction order and if the input files are extracted before the output file, it would trigger a unneccessary frontend build. I think the current solution is the best we can do.

Generally, I think the timestamp approach with `make` is flawed. Something content-hash based may be better, but I guess that's a topic for another PR.